### PR TITLE
Introduce beta example for deploying on DigitalOcean App Platform using 1Password CLI and deprecate current App Platform example

### PR DESCRIPTION
**Pull request by ag-adampike**
This PR introduces a beta deployment example that uses the DigitalOcean shell plugin with 1Password CLI to deploy 1Password SCIM bridge on DigitalOcean App Platform.

It also deprecates the current App Platform deployment example in favour of this one to reduce cost and complexity and increase speed of deployment.

Reasons to use this deployment example are detailed in the README, but to summarize:
- For standard deployments, App Platform will host SCIM bridge for a predictable cost of $10 USD/month (at the time of last review).
- There is no need to set up a DNS record. DigitalOcean automatically provides a URL for SCIM bridge.
- 1Password SCIM bridge deploys directly to DigitalOcean from a local terminal. There is no requirement to clone this repository for this deployment.

The deployment example also serves as a practical example of several `op` commands and deploys in seconds.

### CODE REVIEW

File: beta/do-app-platform-op-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# [Beta] Deploy 1Password SCIM bridge on DigitalOcean App Platform with 1Password CLI
+
+This deployment example describes how to deploy 1Password SCIM bridge as an app on DigitalOcean's [App Platform](https://docs.digitalocean.com/products/app-platform/) service using [1Password CLI](https://developer.1password.com/docs/cli), the DigitalOcean command line interface ([`doctl`](https://docs.digitalocean.com/reference/doctl/)), and the DigitalOcean [1Password Shell Plugin](https://developer.1password.com/docs/cli/shell-plugins/).
```
Comment (accraw) [inline beta/do-app-platform-op-cli/README.md:3]: You can link right to the doctl shell plugin: https://developer.1password.com/docs/cli/shell-plugins/digitalocean

Comment (ag-adampike) [reply beta/do-app-platform-op-cli/README.md:3]: I actually linked to the full page on purpose in this section to show off _all_ the plugins, and include the link directly to the DO shell plugin below:
https://github.com/1Password/scim-examples/blob/5b7f1c95da98e4bbe93415323d564a6a408b81a2/beta/do-app-platform-op-cli/README.md?plain=1#L55

I can revert this if you think it's confusing.

File: beta/do-app-platform-op-cli/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  > **Note**
+  >
+  > Try 1Password Business free for 14 days: <https://start.1password.com/sign-up/business>
+- A DigitalOcean account with available quota for two droplets
+  > **Note**
+  >
+  > If you don't have a DigitalOcean account, you can sign up for a free trial with starting credit: <https://try.digitalocean.com/freetrialoffer/>
+- A Mac or Linux terminal with Bash, Zsh, or Fish
+<!--
+TODO: Investigate Windows Terminal, Powershell, WSL
```
Comment (accraw) [inline beta/do-app-platform-op-cli/README.md:32]: We don't support Windows yet for Shell Plugins

Comment (ag-adampike) [reply beta/do-app-platform-op-cli/README.md:32]: Yeah, I know. This is a note for me to potentially document a workaround for people working from Windows environments (e.g. auth to `doctl` without using the shell plugin).

File: beta/do-app-platform-op-cli/README.md (hunk 1 above)
Comment (accraw) [reply beta/do-app-platform-op-cli/README.md:3]: Nope, that makes sense

File: beta/do-app-platform-op-cli/README.md (hunk 2 above)
Comment (accraw) [reply beta/do-app-platform-op-cli/README.md:32]: Ah, gotcha

**Review by accraw — APPROVED** [review]
The shell plugins side of things looks good to me!

**Comment by ag-adampike** [comment]
> The shell plugins side of things looks good to me!

Thanks for the review, @accraw! 😊

File: beta/do-app-platform-op-cli/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+- [`README.md`](./README.md): the document that you are reading. 👋😃
+- [`op-scim-bridge.yaml`](./op-scim-bridge.yaml): an App Platform [app spec](https://docs.digitalocean.com/glossary/app-spec/) for 1Password SCIM bridge [templated with secret references](https://developer.1password.com/docs/cli/secrets-template-syntax) to load secrets from your 1Password account.
+
+## Overview
+
+Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
+
+- For standard deployments, App Platform will host your SCIM bridge for a predictable cost of $10 USD/month (at the time of last review).
+- You do not need to manage a DNS record. DigitalOcean automatically provides a unique URL for your SCIM bridge.
```
Comment (black-bryan) [inline beta/do-app-platform-op-cli/README.md:17]: We should include a point about how DO handles the certificate management for this DNS record as a benefit as well.

Comment (ag-adampike) [reply beta/do-app-platform-op-cli/README.md:17]: Noted in the latest commit. Thanks for the review!

**Review by scottisloud — DISMISSED** [review]
Tested this personally and ran through it with a customer. After resolving the base64 issue in https://github.com/1Password/scim-examples/pull/222/commits/d7fb420c3cfc07cdc9a8672343ef2529f65bfc49 I think we're in a good place.

File: beta/do-app-platform-op-cli/README.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+<!--
+TODO: Add instructions for vertical scaling
+
+### Scaling
+
+ -->
+
+<!-- TODO: Allow Workspace credentials to be loaded -->
```
Comment (grellyd) [inline beta/do-app-platform-op-cli/README.md:146]: I understand this is a beta project, but I'd prefer to see these represented in the Readme as a more formal todo list.

EG:
```md

## TODO
 
- [ ] Add instructions for vertical scaling
- [ ] Enable the automatic loading of workspace credentials
```

**Review by grellyd — APPROVED** [review]
Our customers are going to love this!

Well written, concise, informative, and frankly I learned a thing or two while reading.

Thank you for putting these changes together folks!

File: beta/do-app-platform-op-cli/README.md (hunk 4 above)
Comment (ag-adampike) [reply beta/do-app-platform-op-cli/README.md:146]: Fair enough. Noted and amended. Thanks for the review!

**Review by grellyd — APPROVED** [review]

**Pull request merged**